django/cantusdb_project/main_app/views/user.py

```diff
@@ -95,20 +100,27 @@ def get_queryset(self):
             .order_by("-date_created")
             .distinct()
         )
+        user = self.request.user
+        sources_with_proofread_permissions = [
+            (source, user_can_proofread_chants_in_source(user, source))
+            for source in my_sources
+        ]
 
-    def get_context_data(self, **kwargs):
-        context = super().get_context_data(**kwargs)
+        user_sources_paginator = Paginator(sources_with_proofread_permissions, 10)
```

Comment:
I see we've replaced `paginate_by = 100` with `Paginator(..., 10)`. Just verifying that this 100 -> 10 substitution is intentional.

Reply:
It was, I thought it made sense to be something lower than 100 for this page but maybe something like 20 or 30 could be best?

Reply:
Paginating by 10 is fine - as long as this change was intended, and not a typo.

For future changes like this, it might make sense to make a separate commit with a message indicating a change has taken place - if there's a bug with pagination in the future, it will be easier to track down the commit that caused the substitution if it has a descriptive commit message.